ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/AbstractCreateStreamCommand.java

```diff
@@ -91,8 +74,8 @@
 
     this.schema = getStreamTableSchema(statement.getElements());
 
-    if (properties.containsKey(DdlConfig.KEY_NAME_PROPERTY)) {
-      final String name = properties.get(DdlConfig.KEY_NAME_PROPERTY).toString().toUpperCase();
+    if (properties.getKey().isPresent()) {
```

Comment:
`KEY` defines the _key field_, so maybe:

```suggestion
    if (properties.getKeyField().isPresent()) {
```